.github/workflows/ci.yaml

```
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - "main"
+  push:
+    branches:
+      - "main"
+
+jobs:
+  build:
```

this job contains lint, fmt, build, so just renaming the job
```suggestion
  CI:
```